Add Lyudmila's profile to the website

lvlugov opened the pull request:
As per title.

benjamincerigo commented on content/data/people.yaml:
```diff
@@ -52,3 +52,12 @@
+  - id: lyudmila
+  fullName: Lyudmila Lugovskaya
+  title: Data Scientist
+  bio: PhD in Finance, University of Cambridge, awarded scholarships. MSc in Psychology. Cross-industry experience of delivering practical data projects. Led data science teams. London/Barcelona based. 
```
There is a white space at the end of this that can be deleted.

lvlugov replied:
423d0c82810d83cb42976d80074648f523105a77

benjamincerigo reviewed (changes requested):
Yipeee! Loving the gif. 

Looks really good and glad that you could get the PR just from the docs in the repo. 

A couple of things that need to be fixed. 

In general, we make the PRs on `staging` rather than main. I think this is missing from the docs. Do you think you could add a line to help the next developer? Maybe [here](https://github.com/datavaluepeople/dvp-landing-static/blob/f9b959090ad21dd9630cf499d2189fead2908a0b/docs/development-deployment.md?plain=1#L6)

lvlugov commented on content/data/people.yaml:
```diff
@@ -52,3 +52,12 @@
   github: willunderwood
   gifPlaceHolder: ../../content/assets/people/will-gif-placeholder.jpg
   profilePictureGif: ../../content/assets/people/will.gif
+  - id: lyudmila
```
70db9268f8579b89dad80371f443698086ffb611

lvlugov commented:
Many thanks for the comments @benjamincerigo! I've made the changes and added a line to the docs on PRs (0aae2ad6055c6084534dcfea58fc0c552968b073). Also changed the base branch to `staging`.

benjamincerigo reviewed (approved):
Great looks good to go. 

If you could:
- merge this PR, this will deploy to the staging website: https://dvp-landing-staging.firebaseapp.com/
- Give staging a check
- if it looks good make a PR from `staging` to `main`
